pkg/liqoctl/wait/wait.go

```diff
@@ -224,10 +225,15 @@ func (w *Waiter) ForConfiguration(ctx context.Context, conf *networkingv1alpha1.
 
 // ForGatewayPodReady waits until the pod of a Gateway resource has been created and is ready.
 func (w *Waiter) ForGatewayPodReady(ctx context.Context, gateway client.Object) error {
-	s := w.Printer.StartSpinner(fmt.Sprintf("Waiting for gateway pod %s to be ready", gateway.GetName()))
-	gatewayDeployment := &appsv1.Deployment{}
+	gatewayDeployment := &appsv1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("gw-%s", gateway.GetName()),
```

Comment:
We should use `forge.GatewayResourceName()` for better compatibility

Reply:
Of course, I thought there was a reason not to have used it before